File: quickstarts/CS/EventBroker/Src/Stoplight/ServiceImplementations/NullLogger.cs
// Copyright (c) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.

using StopLight.ServiceInterfaces;

namespace StopLight.ServiceImplementations
{
	internal class NullLogger : ILogger
	{
		#region ILogger Members

		public void Write(string message)
		{
		}

		#endregion
	}
}

File: quickstarts/VB/Stoplight/Src/ServiceImplementations/NullLogger.vb
' Copyright (c) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.

Imports StopLight.ServiceInterfaces

Namespace StopLight.ServiceImplementations
	Friend Class NullLogger
    Implements ILogger

#Region "ILogger Members"

    Public Sub Write(ByVal message As String) Implements ILogger.Write
    End Sub

#End Region

  End Class
End Namespace
